packages/agreement-process/src/services/readmodel/readModelService.ts

```diff
@@ -551,14 +548,14 @@ export function readModelServiceBuilder(
         },
         {
           $match: {
+            ...makeFilter("consumerId", consumerIds),
+            ...makeFilter("producerId", producerIds),
```

Comment:
Fixing a bug caused by the missing object spread. I moved them as first filters just for consistency with the rest of the queries, where we put `...makeFilter` filters before the rest of the filters defined inline